meta-mender-imx6ul/recipes-core/systemd/systemd-conf/eth.network

```diff
@@ -0,0 +1,9 @@
+[Match]
+Name=eth*
+KernelCommandLine=!nfsroot
+
+[Network]
+DHCP=v4
+
+[DHCPv4]
+UseHostname=false
```

Comment:
This is not Mender related and should not be here. Actually there are similar definitions for this in [meta-mender-demo](https://github.com/mendersoftware/meta-mender/tree/master/meta-mender-demo), so this would just be redundant.